dbNSFP.pm

```diff
@@ -60,12 +60,12 @@ limitations under the License.
  > cat dbNSFP3.5a_variant.chr* | grep -v ^#chr | sort -k1,1 -k2,2n - | cat h - | bgzip -c > dbNSFP.gz
  > tabix -s 1 -b 2 -e 2 dbNSFP.gz
  
- For release 4.0b2 with GRCh38/hg38 data:
- > wget ftp://dbnsfp:dbnsfp@dbnsfp.softgenetics.com/dbNSFP4.0b2a.zip
- > unzip dbNSFP4.0b2a.zip
- > zcat dbNSFP4.0b2a_variant.chr1.gz | head -n1 > h
- > zgrep -h -v ^#chr dbNSFP4.0b2a_variant.chr* | sort -k1,1 -k2,2n - | cat h - | bgzip -c > dbNSFP4.0b2a.gz
- > tabix -s 1 -b 2 -e 2 dbNSFP4.0b2a.gz
+ For release 4.0a with GRCh38/hg38 data:
```

Comment:
Docs could have a note - version 2.9 only supported for GRCh37.

Reply:
Version 2.9 is supported for both GRCh37, GRCh38 as far as I know. Can you give me an example where it is not working? Happy to test it.